src/index.html

```diff
@@ -1,13 +1,73 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="
+            width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0
+          ">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="main">
+      <div class="wrapper">
+        <div class="card" data-qa="card">
+          <div class="card__img">
+
+          </div>
+
+          <div class="card__block-1">
+            <h2 class="card__title">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h2>
+
+            <h3 class="card__subtitle">
+              Код товара: 195434
+            </h3>
```

Comment:
```suggestion
            <p class="card__subtitle">
              Код товара: 195434
            </p>
```

Text 'Код товара:' can be title, but not value '195434'